scripts/pull_github_pr.sh

```diff
@@ -32,22 +32,21 @@ PR_DESCR=$(jq -r .body <<< $PR_DATA)
 PR_REF=$(jq -r .head.ref <<< $PR_DATA)
 PR_LOGIN=$(jq -r .head.user.login <<< $PR_DATA)
 PR_REPO=$(jq -r .head.repo.html_url <<< $PR_DATA)
```

Comment:
Unrelated to this patch, but this line is not needed (and wasn't needed before as well).